src/main/java/com/redhat/lightblue/healthcheck/HealthCheckResource.java

```diff
@@ -78,7 +81,17 @@ public Response health() {
             assertEquals("updated", found.getValue());
 
             DataDeleteRequest deleteRequest = new DataDeleteRequest(ENTITY);
-            deleteRequest.where(Query.withValue("_id", Query.eq, uuid));
+            deleteRequest.where(
+                Query.or(
+                    Query.withValue("_id", Query.eq, uuid),
+                    Query.withValue("creationDate", Query.lte,
+                            Date.from(LocalDateTime
+                                    .now()
+                                    .minusDays(1)
```

Comment:
Change made. I was attempting to handle multiple time zones clobbering each other.